VultisigApp/VultisigApp/Services/Blowfish/BlowfishService.swift

```diff
@@ -11,22 +11,21 @@ import BigInt
 struct BlowfishService {
     static let shared = BlowfishService()
     
-    func scanTransactions
-    (
+    enum BlowfishServiceError: Error {
```

Comment:
Enums within this struct can be moved to an extension of BlowfishService. Makes the more cleaner and structures since enums are not accessed that frequently.